worker/exec/konnector_test.go

```diff
@@ -162,7 +167,7 @@ echo "{\"type\": \"manifest\", \"message\": \"$(ls ${1}/manifest.konnector)\" }"
 			assert.Equal(t, "manifest", doc2.M["type"])
 
 			msg2 := doc2.M["message"].(string)
-			assert.True(t, strings.HasPrefix(msg2, t.TempDir()))
+			assert.True(t, strings.HasPrefix(msg2, "/tmp"))
```

Comment:
You can use `os.TempDir()` instead. It was the value before the (incorrect) migration to `t.TempDir()`